File: cc practice/intvs.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int t;
    cin>>t;
    while(t--) {
        int n, k;
        cin>>n>>k;
        int arr[n];
        int correct=0;
        int slow=0;
        int bot=0;
        int least = ceil(n/2);
        for(int i=0; i<n; i++) {
            cin>>arr[i];

            if(arr[i]!=-1) {
                correct++;
            }
            else if(arr[i]>k) {
                slow++;
            }
            else if(arr[i]<=1) {
                bot++;
            }
        }
        cout<<least<<" ";
        cout<<correct<<" ";
        cout<<slow<<" ";
        cout<<bot<<" ";
        if(correct>=least) {
            if(slow<1) {
                if(bot!=n) {
                    cout<<"Accepted"<<"\n";
                }
                else {
                    cout<<"Bot"<<"\n";
                }
            } 
            else {
                cout<<"Too Slow"<<"\n";
            }
        }
        else {
            cout<<"Rejected"<<"\n";
        }
    }
}

File: cc practice/intvs.py
from math import ceil
t = int(input())
for _ in range(t):
    n, k=map(int, input().split())
    arr=[int(x) for x in input().split()]
    correct=0
    slow=0
    bot=True
    least=ceil(n/2)
    for time in arr:
        if time != -1:
            correct+=1
        if time > k:
            slow += 1
        if time > 1:
            bot = False
    # print(least, correct, time, bot)


    if correct>=least:
        if slow<1:
            if ((correct==n) and bot):
                print("Bot")
            else:
                print("Accepted")
        else:
            print("Too Slow")
    else:
        print("Rejected")